### PR TITLE
Update to PHP 7.2 + Vendors

### PR DESCRIPTION
**Pull request by albert-gonzalez**
Hi!

In order to have the dependencies of our project updated, we have forked your library and updated it. It seems that you don't maintain this repo anymore, but it would be interesting if you generated a new version with these changes.

PHP 7.2 is required because older versions are not maintained anymore. More info: https://www.php.net/supported-versions.php

# Changes
* Docker + docker-compose configured
* Script to execute tests in docker
* Vendors updated
  * PHPUnit 8.1
  * UUID 3.8
  * rmccue/requests 1.7
  * php-coveralls 2.1
* PHP 7.2 Required

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://github.com/snowplow/snowplow/wiki/CLA to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

**Comment by albert-gonzalez** [comment]
I signed it!

**Comment by snowplowcla** [comment]
@albert-gonzalez  has signed the Software Grant and Corporate Contributor License Agreement

File: .coveralls.yml [hunk 1]
```diff
@@ -1 +0,0 @@
-src_dir: .
```
Comment (BenFradet) [inline .coveralls.yml]: do we still need this file?

Comment (albert-gonzalez) [reply .coveralls.yml]: No, I removed it and Travis still passes. Thank you!

**Comment by BenFradet** [comment]
@mhadam @oguzhanunlu could you have a look? thanks.

**Comment by albert-gonzalez** [comment]
Hi!

If I have to do something else, please tell me :)

**Comment by oguzhanunlu** [comment]
Looks good to me! Thanks @albert-gonzalez !

**Comment by oguzhanunlu** [comment]
Closing the PR since we incorporated the work into #75 .
Thanks again @albert-gonzalez , your contribution is cherry-picked into the release branch.

**Comment by albert-gonzalez** [comment]
Glad to hear it! Thanks for your great work!

**Pull request closed without merging**